src/libpgmoneta/utils.c

```diff
@@ -1892,6 +1893,19 @@ copy_file(void* arg)
    int saved_errno = -1;
    int permissions = -1;
    struct worker_input* fi = NULL;
+   struct configuration* config = (struct configuration*)shmem;
+   struct token_bucket* bucket = NULL;
+
+   // default is 0
+   if (config->restore_max_rate)
```

Comment:
Still here

Reply:
Should we remove the restore functionality for now?

Reply:
Yes, we need a discussion about it

Reply:
Feel free to start this after this PR is in